spec/dummy/client/app/components/HelloWorld.jsx

```diff
@@ -10,12 +11,12 @@ class HelloWorld extends React.Component {
   }
 
   _handleChange() {
-    const name = React.findDOMNode(this.refs.name).value;
+    const name = Common.reactFindDOMNode(this.refs.name).value;
```

Comment:
I think this is just for testing, so it's not critical. The https://github.com/shakacode/react-webpack-rails-tutorial/ is geared toward a complete example.